entity-cluster-bot/src/persistance.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import pickle
+import os
+import sys
+import forta_agent
+from datetime import datetime
+import bz2
+import networkx as nx
+import time
+
+
+from boto3.dynamodb.conditions import Key, Attr
+
+
+try:
+    from src.keys import AWS_ACCESS_KEY
```

Comment:
we would want to put all your secrets keys into a secrets json, which you can access using jwt. I can work with steven on that as we probably want to setup the infra under the forta foundation account.

Reply:
done